test/runtests.jl

```diff
@@ -18,17 +30,46 @@ using Test, BFloat16s, Printf, Random
     @test BFloat16(2)   != BFloat16(1)
     @test BFloat16(2f0) != BFloat16(1f0)
     @test BFloat16(2.0) != BFloat16(1.0)
+    @test BFloat16(NaN) != BFloat16(1.0)
+    @test !(BFloat16(1.0) == BFloat16(NaN))
     @test iszero(BFloat16(0)) == true
     @test iszero(BFloat16(3.45)) == false
 end
 
+@testset "trunc" begin
+    bf_val = BFloat16(5.5)
+    @testset "$Ti" for Ti in (Int8, Int16, Int32, Int64, Int128, UInt8, UInt16, UInt32, UInt64, UInt128)
+        if !(BFloat16s.llvm_arithmetic && sizeof(Ti) == 16) # Can't use skip  argument in @test on 1.6
```

Comment:
This is only here until #94 is merged. I just didn’t want to have one depend on the other.